curriculum/challenges/english/07-scientific-computing-with-python/learn-classes-and-objects-by-building-a-sudoku-solver/6606bf4561f8794a0d345919.md

```diff
@@ -0,0 +1,59 @@
+---
+id: 6606bf4561f8794a0d345919
+title: Step 27
+challengeType: 20
+dashedName: step-27
+---
+
+# --description--
+
+It's time to test the `valid_in_row` method. Call `valid_in_row` on `gameboard`. Pass it `0` and `8` as the arguments and print the result.
+
+Again, note how the method is defined with three parameters, but it is called only with two arguments, since `self` is automatically passed in as the object the method is called on.
```

Comment:
```suggestion
Again, note how the method is defined with three parameters, yet it is called with only two arguments because `self` is automatically passed as the object on which the method is called.
```